modules/collect_results_add_metadata.nf

```diff
@@ -7,6 +7,7 @@ process collect_results_final {
     file 'transformed_consensus.tsv'
     file 'domain_globularity.tsv'
     file 'domain_plddt_and_lur.tsv'
+    file 'All_taxonomy.tsv'
```

Comment:
is there are reason for the uppercase 'A' in the filename? (if not, then it's best to have consistent capitalisation throughout the code)